tasks/system.yml

```diff
@@ -24,6 +24,10 @@
       - libwebp-dev
   tags: apt
 
+- name : Update all packages
+  apt: name="*" state=latest update_cache=true force_apt_get=true
```

Comment:
Instead of adding this to update all packages, I'm wondering if we could achieve the same thing by adding `state: latest` to the _install required image libraries_ and _install python3 dependencies_ tasks? The reason I ask is that it looks like the [apt module[(https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html#parameter-state) documentation says if you omit `state` it defaults to `present`, which might explain why those packages aren't being updated on the system.

Reply:
That sounds like the better way to go about this and it looks like we can also add `update_cache` as well!